src/modules/views/Public/PublicFolderView.jsx

```diff
@@ -176,70 +177,76 @@ const PublicFolderView = () => {
   const isAddToMyCozyFabDisplayed = isMobile && isPreview && isShareNotAdded
 
   return (
-    <Main isPublic={true}>
-      <ModalStack />
-      <ModalManager />
-      {isSharingBannerPluginDisplayed && <SharingBannerPlugin />}
-      <Content className={isMobile ? '' : 'u-ml-1 u-pt-1'}>
-        <FolderViewHeader>
-          {currentFolderId && (
-            <>
-              {isOldBreadcrumb ? (
-                <OldFolderViewBreadcrumb
-                  currentFolderId={currentFolderId}
-                  getBreadcrumbPath={geTranslatedBreadcrumbPath}
+    <Dropzone
+      role="main"
```

Comment:
`role` is not spread anymore, you can remove it. Why did you add it? (I'm asking because maybe you picked an old part of code as an example and we need to update it also)